cwl/src/main/scala/cwl/CwltoolRunner.scala

```diff
@@ -0,0 +1,71 @@
+package cwl
+
+import ammonite.ops.ImplicitWd._
+import ammonite.ops._
+import better.files.{File => BFile}
+import com.typesafe.config.ConfigFactory
+import org.broadinstitute.heterodon.ExecAndEval
+
+/**
+  * Interface for running cwltool.
+  */
+sealed trait CwltoolRunner {
+  def salad(file: BFile): String
+}
+
+object CwltoolRunner {
+  private lazy val config = ConfigFactory.load
+
+  lazy val instance: CwltoolRunner = {
+    val runnerClass = config.getString("cwltool-runner.class")
+    Class.forName(runnerClass).newInstance().asInstanceOf[CwltoolRunner]
```

Comment:
ToL Could just import this lib/class and `new` it up, no need for jvm-level instantiation IMO